### PR TITLE
fix(client): profile card background

### PR DESCRIPTION
**Pull request by huyenltnguyen**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

We're using `gray00` as the background color of the profile cards, and this is causing the text to be unreadable in dark theme.

<details>
<summary>Screenshot</summary>

Screenshot 2024-12-14 at 05 28 16
</details>

I'm changing the variable to `primary-background` instead, so that the color can change based on the selected theme.

| Light | Dark |
| --- | --- |
| Screenshot 2024-12-14 at 05 45 57 | Screenshot 2024-12-14 at 05 45 46 |

### CODE REVIEW

**Comment by Sembauke** [comment]
Whoops

**Review by Sembauke — APPROVED** [review]

**Comment by ahmaxed** [comment]
Thanks for catching that. let's go for --secondary-background to be consistent with the map pages.

File: client/src/components/profile/profile.css [hunk 1]
```diff
@@ -8,7 +8,7 @@
   margin: 0;
   padding: 1em;
   box-shadow: none;
-  background: var(--gray00);
+  background: var(--primary-background);
```
Comment (ahmaxed) [inline client/src/components/profile/profile.css:11]:
```suggestion
  background: var(--secondary-background);
```

**Review by ahmaxed — APPROVED** [review]

**Comment by ahmaxed** [comment]
The card borders also needs fixing I believe.

**Comment by ahmaxed** [comment]
Screenshot 2024-12-15 at 12 49 33 PM

**Pull request merged**